### PR TITLE
Apply regex search algorithm

### PR DESCRIPTION
**Pull request by andhikaribrahim**
#### Proof of Concept
Screenshot from 2019-10-19 03-23-47

#### Issue
#1

### CODE REVIEW

**Comment by abhishek3008** [comment]
Hi andhika ... Thanks for contributing
This is correct when user searches for 25V. 
I would like to add what if the user search for 25 V?... then?
Because according the user he might not know how product names are.. So we have to take care of this also.

**Review by abhishek3008 — COMMENTED** [review]
Hi andhika ... Thanks for contributing
This is correct when user searches for 25V.
I would like to add what if the user search for 25 V?... then?
Because according the user he might not know how product names are.. So we have to take care of this also.

**Comment by andhikaribrahim** [comment]
Ok, i'll look into that later.

File: index.php [hunk 1]
```diff
@@ -25,24 +24,34 @@
...
 
-	<?php
+	<div>Search Results</div>
+	<div>
+		<?php 
+			$input = $_POST['searchbox'];
+
+			$results = array_map(function ($item) use ($input) {
+			    $input = strtolower($input);
+			    $lowerItem = strtolower($item);
```
Comment (andhikaribrahim) [inline index.php]:
```suggestion
			    $lowerItem = str_replace(" ", "", strtolower($item));
```

File: index.php [hunk 2]
```diff
@@ -25,24 +24,34 @@
...
 	</div>
 
-	<?php
+	<div>Search Results</div>
+	<div>
+		<?php 
+			$input = $_POST['searchbox'];
+
+			$results = array_map(function ($item) use ($input) {
+			    $input = strtolower($input);
```
Comment (andhikaribrahim) [inline index.php]:
```suggestion
			    $input = str_replace(" ", "", strtolower($input));
```

**Comment by andhikaribrahim** [comment]
@abhishek3008 Hi, I updated the PR. 
Please check again. thanks.

**Pull request merged**